src/components/Movies/Movies.js

```diff
@@ -12,18 +12,16 @@ const Movies = ({ movies, displayMovieDetail }) => {
         title={movie.title}
         id={movie.id}
         key={movie.id}
-        displayMovieDetail={displayMovieDetail}
       />
     ));
   } else {
     for (let i = 0; i < 5; i++) {
-      movieCards.push(<div className="movie" key={i}><div className="movie__poster--placeholder"></div></div>);
+      movieCards.push(<div className="movie" key={i}><div className="movie__poster--placeholder" /></div>);
```

Comment:
I see what you did here but the end div is still there

Reply:
There were actually two divs: a div within a div. For some reason it only worked to make one of them self-closing.